### ISSUE TITLE
Валит сервер

### ISSUE THREAD

**Issue by daaner** [LaravelRUS/SleepingOwlAdmin#1193]
Многократное нажатие кнопки создать (более 100 раз) уложило мой сервер
Неплохо бы проверить и заблокировать кнопку после клика
будет время - проверю

**Comment by daaner** [comment]
100% проца 100% по памяти. Хз что было, но мне не понравилось

**Comment by sensor-dream** [comment]
Все просто, есть такой баг увсех однако смотрелок, создай тестилку и посмотри. А еще лучше, как я дагадуовуоесь, пользуешся vscode при разработке, баги лнут к MS Ж)  09.05.2020, 03:08, "Andrey" <notifications@github.com>: 100% проца 100% по памяти. Хз что было, но мне не понравилось—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.  Pavel M. Teslenkosensor-dream@sensor-dream.ru

**Comment by vkn999** [comment]
> Многократное нажатие кнопки создать (более 100 раз) уложило мой сервер
> Неплохо бы проверить и заблокировать кнопку после клика
> будет время - проверю

Последняя версия ветки dev? Правильно понимаю, что не встроенный вэб-сервер лары, а тестовый/боевой? Под создать понимается SaveAndCreate()?

**Comment by daaner** [comment]
> > Многократное нажатие кнопки создать (более 100 раз) уложило мой сервер
> > Неплохо бы проверить и заблокировать кнопку после клика
> > будет время - проверю
> 
> Последняя версия ветки dev? Правильно понимаю, что не встроенный вэб-сервер лары, а тестовый/боевой? Под создать понимается SaveAndCreate()?

Да, боевой. Много раз тыкаешь или сейв или сейвкрейт... Нужно лочить кнопки, думаю

**Comment by daaner** [comment]
> Все просто, есть такой баг увсех однако смотрелок, создай тестилку и посмотри. А еще лучше, как я дагадуовуоесь, пользуешся vscode при разработке, баги лнут к MS Ж)  09.05.2020, 03:08, "Andrey" <notifications@github.com>: 100% проца 100% по памяти. Хз что было, но мне не понравилось—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.  Pavel M. Teslenkosensor-dream@sensor-dream.ru

Во-первых, какая разница чем я пользуюсь при разработке (Атом юзаю)
Во-вторых, это проблема очень далека от ИДЕ
